.github/workflows/trivy.yaml

```
@@ -63,11 +66,10 @@ jobs:
           -t local-image:${{ github.sha }} \
           .
 
-      # 3. Vulnerability scan with Trivy
       - name: Run Trivy vulnerability scanner
-        uses: aquasecurity/trivy-action@master   # pin to the latest tag (Mar 14 2025)
+        uses: aquasecurity/trivy-action@master
```

```suggestion
        uses: aquasecurity/trivy-action@0.30.0
```
Please pin to the latest version